### PR TITLE
termbox: init at 1.1.2

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by fgaz** [comment]
@edude03 only the build system is in python. The actual library is C

File: pkgs/development/libraries/termbox/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    sha256 = "08yqxzb8fny8806p7x8a6f3phhlbfqdd7dhkv25calswj7w1ssvs";
+  };
+  nativeBuildInputs = [ python3 ];
+  configurePhase = "python3 ./waf configure --prefix=$out";
+  buildPhase = "python3 ./waf build";
+  installPhase = "python3 ./waf install --destdir=$out";
+  meta = with stdenv.lib; {
+    description = "Library for writing text-based user interfaces";
+    license = licenses.mit;
+    homepage = "https://github.com/nsf/termbox#readme";
```
Comment (timokau) [inline pkgs/development/libraries/termbox/default.nix]: Quotes around URLs are not necessary in nix.

**Review by timokau — APPROVED** [review]
Thanks :)

File: pkgs/development/libraries/termbox/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/development/libraries/termbox/default.nix]: I always forget about that >.<

**Pull request merged**